modules/order/commerce_order.module

```diff
@@ -80,6 +84,13 @@ function template_preprocess_commerce_order(&$variables) {
   foreach (Element::children($variables['elements']) as $key) {
     $variables['order'][$key] = $variables['elements'][$key];
   }
+
+  if ($order->getBillingProfile()) {
+    $variables['order']['billing_information'] = $order->getBillingProfile()->get('address')->view([
```

Comment:
Why would we not render the entire billing profile? Doing it this way means no phone field / VAT number is shown even if the fields are attached.